src/pages/chaeyeon/Main/Article.js

```diff
@@ -0,0 +1,148 @@
+        </div>
+        <div class="feed_picture">
+          <img src="images/chaeyeon/candy2.jpeg" alt="피드 사진" />
+        </div>
+        {/* <!--feedbottom--> */}
+        <div className="feed_bottom">
+          <div className="emoticon_box">
+            <div className="emoticon_box2">
+              <div className="heart_box">
+                <img
+                  className="heart"
+                  src="images/chaeyeon/like_good.png"
+                  alt="하트"
+                />
+              </div>
+              <div className="comment_box">
+                <img
+                  className="coment"
+                  src="images/chaeyeon/comment.PNG"
+                  alt="코멘트"
+                />
+              </div>
+              <div className="direct_box">
+                <img
+                  className="direct"
+                  src="images/chaeyeon/공유.png"
+                  alt="공유"
+                />
+              </div>
+            </div>
+            <div className="bookmark_box">
+              <img
+                className="bookmark"
+                src="images/chaeyeon/save.PNG"
+                alt="저장"
+              />
+            </div>
+          </div>
+        </div>
+        {/* <!--/feedbottom--> */}
+        <div className="feed_like_box">
+          <div className="feed_like_picture">
+            <img
+              className="feed_like_peolpe"
+              src="images/chaeyeon/peple01.png"
+              alt="캔디"
+            />
+          </div>
+          <div className="feed_like">coco님 외 100명이 좋아합니다.</div>
+        </div>
+        <div className="feed_article">
+          <div className="feed_article_box">
+            <div className="comments_container">
+              <div className="comments">캔디가 제일 이쁨❤️</div>
+
+              <button onClick={HandleEdit}>✎</button>
+
+              <div className="comments1">더 보기</div>
+            </div>
+            <div className="comments1">
+              댓글 <span id="count"></span> 모두 보기
+            </div>
+            <div className="comments1_box">
+              {/* // 똑같은 형식으로 올려지는 originalComments-배열반복 / 새로운 comment가 변수로 올라감 */}
+              {originalComments.map(comment => (
+                <Comment comment={comment} />
+                // originalComments올라가는 모양
+              ))}
+            </div>
+
+            <div className="new_comments"></div>
+            <div className="comments2">2일 전</div>
+          </div>
+        </div>
+        <div className="inputContainer">
+          <div className="type_comment">
+            <input
+              className="inputBox"
+              type="text"
+              placeholder="댓글 달기..."
+              value={comment}
+              onChange={HandleInput}
+              onKeyUp={HandleEnter}
+            />
+          </div>
+          <span>
+            {/* 게시누르면 올라가는 장소와 방법, 인픗값 발생 */}
+            <button onClick={HandleComment} className="buttonBox" type="summit">
```

Comment:
- 오타 주의
- `type="submit"`은 default 값이기 때문에 명시할 필요 없음!
- 오히려 form 태그 안에서 submit type이 부여되어 있으면 form을 제출하면서 새로고침하는 동작을 취하게 되기 때문에, `type="button"`을 명시해주어야 함
```suggestion
            <button onClick={HandleComment} className="buttonBox" type="submit">
```